pkg/katautils/config.go

```diff
@@ -600,10 +598,7 @@ func updateRuntimeConfigProxy(configPath string, tomlConf tomlConfig, config *oc
 	}
 
 	for k, proxy := range tomlConf.Proxy {
-		switch k {
-		case ccProxyTableType:
-			config.ProxyType = vc.CCProxyType
-		case kataProxyTableType:
+		if k == kataProxyTableType {
```

Comment:
So two things after talking with @gabibeyer, we don't expect more than one proxy definition, so we should check the number of entries provided through the config. Also, we don't expect to support multiple types of `proxy`, `shim` or `agent`, so we should simply have a toml entry this way:
```toml
[proxy]
path = "foo"

[shim]
path = "bar"
```
And to even go one step further, we can remove the agent entry itself since that was a simple placeholder to determine the type of agent.

/cc @jodh-intel @amshinde @egernst

Reply:
I prefer we keep original ones and don't change it.  Modify existing field names can break compatibility and make me anxious, `proxy.kata` doesn't harm :)

Reply:
If we do this I think we should maintain backward compatibility here to allow runtime to be used with older toml possibly in `/etc/kata-containers`.

Reply:
Sounds good! I kept the old formatting, then added a 'default' case statement that returns an error is the type passed is not supported.